docs/release-notes/NuGet-6.3.md

```diff
@@ -0,0 +1,125 @@
+---
+title: NuGet 6.3 Release Notes
+description: Release notes for NuGet 6.3 including new features, bug fixes, and DCRs.
+author: <GithubAlias>
+ms.author: <MicrosoftAlias>
+ms.date: 8/2/2022
+ms.topic: conceptual
+---
+
+# NuGet 6.3 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**<NuGetVersion>**](https://nuget.org/downloads) | [Visual Studio <VSYear> version <VSVersion>](https://visualstudio.microsoft.com/downloads/) | [<SDKVersion>](https://dotnet.microsoft.com/download/dotnet-core/<SDKMajorMinorVersionOnly>)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio <VSYear> with.NET Core workload
+
+## Summary: What's New in 6.3
+
+* Implement "consuming pdbs from packages in PackageReference" - [#11352](https://github.com/NuGet/Home/issues/11352)
```

Comment:
@heng-liu Would you be able to help us with a small description of this work for a feature summary?

Reply:
Given that this has no functional impact without the SDK change, should we `save` this for next release?

Reply:
Removing this for next release @JonDouglas @nkolev92